File: Easy/Euclids-algorithm/Euclids-algorithm.js
var inputs = readline().split(' ');
let a = parseInt(inputs[0]), ao = a;
let b = parseInt(inputs[1]), bo = b;
while (1) {
    if (b == 0) break;
    console.log(a + "=" + b + "*" + (a / b >> 0) + "+" + a % b);
    a %= b;
    if (a == 0) break;
    console.log(b + "=" + a + "*" + (b / a >> 0) + "+" + b % a);
    b %= a;
}
console.log("GCD(" + ao + "," + bo + ")=" + Math.max(a, b));

File: Easy/Euclids-algorithm/Euclids-algorithm.java
import java.util.*;
class Solution {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);
        int a = in.nextInt(), ao = a;
        int b = in.nextInt(), bo = b;
        while (true) {
            if (b == 0) break;
            System.out.println(a + "=" + b + "*" + a / b + "+" + a % b);
            a %= b;
            if (a == 0) break;
            System.out.println(b + "=" + a + "*" + b / a + "+" + b % a);
            b %= a;
        }
        System.out.println("GCD(" + ao + "," + bo + ")=" + Math.max(a, b));
    }
}
